src/js/nightly.js

```diff
@@ -108,7 +108,7 @@ function buildNightlyHTML(files) {
     var nameOfFile = (eachAsset.binary_name);
     var uppercaseFilename = nameOfFile.toUpperCase(); // make the name of the file uppercase
     NIGHTLYOBJECT.thisPlatform = findPlatform(eachAsset); // get the searchableName, e.g. MAC or X64_LINUX.
-    var type = nameOfFile.includes('-jre') ? 'jre' : 'jdk';
+    var type = nameOfFile.indexOf('-jre') !== -1 ? 'jre' : 'jdk';
```

Comment:
Can you add a comment as to why (e.g. IE is being difficult)

Reply:
Do you mean to the code?  The new line is fairly standard JS, so that might not be worthwhile.  The need for this change is due to IE not supporting `includes`, preventing IE users (thinking corporate settings mostly) from using the Nightly page.

Reply:
Yeah, I was thinking of adding the comment for IE support, else someone will revert your change down the line...

Reply:
If that's a concern it might be better to just include a polyfill for `includes`, or the aforementioned `@babel/polyfill` to support everything.